tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -1,84 +1,52 @@
 import logging
 import pytest
 
-from ocs_ci.framework.testlib import ManageTest, tier1, tier3, acceptance
+from ocs_ci.framework.testlib import ManageTest, tier1, acceptance
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.exceptions import UnexpectedBehaviour
 from ocs_ci.ocs.resources import pod
 from ocs_ci.utility.retry import retry
 from tests import helpers
-from tests.fixtures import (
-    create_ceph_block_pool, create_rbd_secret, create_cephfs_secret,
-    create_project
-)
 
 logger = logging.getLogger(__name__)
 
 
-class BaseDynamicPvc(ManageTest):
+class TestDynamicPvc(ManageTest):
     """
-    Base class for Dynamic PVC creation tests
+    Automates the following test cases:
+    OCS-530 - RBD Based RWO Dynamic PVC creation with Reclaim policy set to Retain
+    OCS-533 - RBD Based RWO Dynamic PVC creation with Reclaim policy set to Delete
+    OCS-525 - CephFS Based RWO Dynamic PVC creation with Reclaim policy set to Retain
+    OCS-526 - CephFS Based RWO Dynamic PVC creation with Reclaim policy set to Delete
+    OCS-542 - CephFS Based RWX Dynamic PVC creation with Reclaim policy set to Retain
+    OCS-529 - CephFS Based RWX Dynamic PVC creation with Reclaim policy set to Delete
     """
-    access_mode = None
-    storage_type = None
-    expected_pod_failure = None
-    expected_pvc_failure = None
+    pvc_size = 10  # size in Gi
 
-    pvc_size = '10Gi'
-    io_size = '512M'
-
-    def dynamic_pvc_base(self, interface_type, reclaim_policy):
+    @pytest.fixture()
+    def setup(
+        self, interface_type, reclaim_policy, storageclass_factory
+    ):
         """
-        Base function for Dynamic PVC creation tests
-        Fetches the worker nodes name list, creates StorageClass and PVC
+        Creates storage class with specified interface and reclaim policy.
+        Fetches all worker nodes
```

Comment:
plz add function out put to description
also, add blank line at the end

Reply:
Updated the docstring